src/Hng.Web/Repo/Interface/IUserRepository.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Hng.Domain.Models;
+
+namespace Hng.Web.Repo.Interface
+{
+    public interface IUserRepository : IGenericRepository<User>
+    {
+        Task<object> GetUserById(int id);
```

Comment:
@davideshett Why are you returning `object` when this is a `UserRepository` and your method name clearly says `GetUserById`?